py_jama_rest_client/client.py

```diff
@@ -710,6 +734,37 @@ def post_item(self, project, item_type_id, child_item_type_id, location, fields,
         JamaClient.__handle_response_status(response)
         return response.json()['meta']['id']
 
+    def post_comment(self, item_id: int, message: str, in_reply_to: int = None):
```

Comment:
Need param to post to item or project.  we should also add a field for comment type